SPECS/xcp-ng-release.spec

```diff
@@ -140,6 +147,15 @@ Obsoletes:      xen-livepatch < 2.0-1.1
 ### xenserver-release-config is only included in real installs
 #Requires:	xenserver-config-packages
 Requires:	python3-xcp-libs
+
+%if %{with use_openssl3}
+# CA-401322: /sbin/update-issue calls openssl command which must link to
+# compatible (or same version) openssl-libs. Here specify the require to
+# ensure openssl-libs, openssl and %%name-config rpms to be updated in order.
+Requires:       openssl >= 3.0.9
+Requires(pre): sed
+Requires(pre): coreutils
```

Comment:
I'm not sure how this gets the packages updated in the order they want, but I haven't given it much thought.

Reply:
The comment only apply to the 1st dep at:

https://github.com/xcp-ng-rpms/xcp-ng-release/pull/33/commits/27fd57ac10c46ceab25ff01815d5e6d7893a0b7f#diff-ff0cb122a768fd67f0299827b8f65762c64492394b3936e0cc649b64b556668aR115

Maybe a newline is missing just after SSL3